doc/modules/feature_selection.rst

```diff
@@ -131,6 +131,26 @@ number of features.
       elimination example with automatic tuning of the number of features
       selected with cross-validation.
 
+.. _select_from_model:
+
+Feature selection using SelectFromModel
+=======================================
+
+:class:`SelectFromModel` is a meta-transformer that can be used along with any
+estimator that has a ``coef_`` or ``feature_importances_`` attribute after fitting.
+It should be given a threshold parameter below which the features are considered
+unimportant and removed. If one has no idea of the prior value of the threshold,
+string inputs like "mean" or "median" or even values like "0.1*mean" can be given
+which SelectFromModel parses internally.
+
+For examples on how it is to be used refer to the sections below.
+
+.. topic:: Examples
+
+    * :ref:`example_feature_selection_select_from_model.py`: Selecting the two
+      most important features from the Boston dataset without knowing the
+      threshold beforehand.
+
 
 .. _l1_feature_selection:
```

Comment:
You should probably just merge the "L1-based feature selection" and "Selecting non-zero coefficients" as a subheading of "Feature selection using SelectFromModel"